### ISSUE TITLE
Breakpoint rules. Add option "Only for remote devices"

### ISSUE THREAD

**Issue by almaz010891** [ProxymanApp/Proxyman#1883]
## Description

I am mobile QA, so I need to intercept requests only from remote devices. But now I can't configure rule for this case

## Why this feature/change is important?

Improve Breakpoint feature usability for remote devices

**Comment by NghiaTranUIT** [comment]
May I ask:
- Have you installed the certificate to your iOS or Android device? You can follow this docs: https://docs.proxyman.io/debug-devices/ios-device

**Comment by almaz010891** [comment]
Yes, of course. I would like to intercept ANY request using Breakpoint functionality, but only from remote devices, requests from local machine that matches rule may be ignored. So additional checkbox "Only for remote devices" for Breakpoint rule may be useful
